db/db_impl.cc

```diff
@@ -1911,6 +1918,16 @@ bool DBImpl::GetIntPropertyInternal(ColumnFamilyData* cfd,
   }
 }
 
+bool DBImpl::HandleImmutableOptionStatistics(std::string* value) {
+  assert(value != nullptr);
+  auto dbstats = immutable_db_options_.statistics.get();
```

Comment:
`auto` is not suggested in the usage like this. See https://google.github.io/styleguide/cppguide.html#auto